podcasts/New Detail/Episode Add/ManualPlaylistsChooserViewController.swift

```diff
@@ -158,15 +158,24 @@ class ManualPlaylistsChooserViewController: PCViewController {
 
         FileLog.shared.console("Added \(added), removed \(removed)")
 
+        var changedPlaylists: Set<EpisodeFilter> = []
+
         manualPlaylists.forEach { playlist in
             if added.contains(playlist.uuid) {
                 dataManager.add(episodes: [episode], to: playlist)
+                changedPlaylists.insert(playlist)
```

Comment:
As the delete method below, would it make sense to save the playlist each time a batch of episodes is added? Wdyt?